scene/3d/label_3d.cpp

```diff
@@ -685,6 +695,38 @@ TextServer::Direction Label3D::get_text_direction() const {
 	return text_direction;
 }
 
+void Label3D::set_localize_numeral_system(bool p_enable) {
+	ERR_MAIN_THREAD_GUARD;
+	if (p_enable == localize_numeral_system) {
+		return;
+	}
+
+	localize_numeral_system = p_enable;
+
+	notification(MainLoop::NOTIFICATION_TRANSLATION_CHANGED);
+}
+
+bool Label3D::is_localizing_numeral_system() const {
+	ERR_READ_THREAD_GUARD_V(false);
+	return localize_numeral_system;
+}
+
+void Label3D::set_auto_translate(bool p_enable) {
+	ERR_MAIN_THREAD_GUARD;
```

Comment:
Threads guards aren't used in `Label3D` currently, so this one should likely be removed (same in the getter). I don't know if it means that the class is thread-safe or unsafe, but at least it operates under different assumptions than Controls.